xc7/utils/prjxray_arch_import.py

```diff
@@ -557,7 +557,10 @@ def main():
             "R": "551",
             "Cin": ".77e-15",
             "Cout": "4e-15",
-            "Tdel": "6.8e-12",
+            # TODO: This value should be the "typical" pip switch delay from
```

Comment:
Can you a log an issue about this?

Reply:
https://github.com/SymbiFlow/symbiflow-arch-defs/issues/762